### PR TITLE
Test CLI example scripts

### PR DESCRIPTION
**Pull request by webmaster128**
This ensures the examples still compile and run

### CODE REVIEW

**Comment by webmaster128** [comment]
In this PR I added `--init` with URLs. I tested `./bin/cosmwasm-cli --debug --init https://raw.githubusercontent.com/CosmWasm/cosmjs/v0.22.0-alpha.1/packages/cli/examples/generate_address.ts`, which works.

```sh
npx @cosmjs/cli --init https://raw.githubusercontent.com/CosmWasm/cosmjs/v0.22.0-alpha.1/packages/cli/examples/generate_address.ts
```

should work as well.

File: .circleci/config.yml [hunk 1]
```diff
@@ -139,6 +139,18 @@ jobs:
           environment:
             SKIP_BUILD: 1
           command: yarn selftest
+      - run:
+          name: Run CLI examples
+          working_directory: packages/cli
+          environment:
+            SKIP_BUILD: 1
+          command: |
+            ./bin/cosmwasm-cli --init examples/delegate.ts --code "process.exit(0)"
```
Comment (ethanfrey) [inline .circleci/config.yml:148]: good to test this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## HEAD
 
+- @cosmjs/cli: Now supports HTTPs URLs for `--init` code sources.
```
Comment (ethanfrey) [inline CHANGELOG.md:5]: This is a great feature add.

Do you want to test that one time in the CI? Or would it be too hard to find some stable code to test against? 

I mean, I guess generate_address should be relatively stable and we could point to master until some PR needs to point to it's own branch?

Comment (webmaster128) [reply CHANGELOG.md:5]: Yeah, testing against an external resource is a problem. I'd leave it as is and use a local HTTP server is we really have problem with this.

**Pull request merged**